packages/samples/headless-react/src/components/category-facet/category-facet.class.tsx

```diff
@@ -122,6 +123,10 @@ export class CategoryFacet extends Component<CategoryFacetProps> {
   }
 
   render() {
+    if (!this.state) {
```

Comment:
This condition is now necessary to avoid an error on the first `render`. Returning `null` here won't make any difference visually since the next render is going to be called immediately.